docs/unfreefeatures.rst

```diff
@@ -189,3 +189,26 @@ Exit code to shell
 The ``self.exit_code`` attribute of your ``cmd2`` application controls
 what exit code is sent to the shell when your application exits from
 ``cmdloop()``.
+
+
+Asynchronous printing
+=====================
+``cmd2`` provides two functions to provide asynchronous feedback to the user without interfering with
```

Comment:
This PR also added a 3rd method which should probably be documented here for setting the window title

Reply:
Added set_window_title()